server/config/persist_options.go

```diff
@@ -508,6 +508,15 @@ func (o *PersistOptions) IsLocationReplacementEnabled() bool {
 	return o.GetScheduleConfig().EnableLocationReplacement
 }
 
+// GetHotRegionSplitSize returns the hot region split size.
+func (o *PersistOptions) GetHotRegionSplitSize() int64 {
```

Comment:
Rename to `GetMaxMovableHotPeerSize` please.

Reply:
fix